builder/copy_linux_test.go

```diff
@@ -0,0 +1,22 @@
+package builder
```

Comment:
You need to revert this to the non-constrained file name and add a build constraint to exclude windows.

```
+build !windows
```

What you have here will break test execution on Darwin.

Reply:
@johnmccabe thanks